plugin/web/components/UI/Icon/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { Image } from "antd";
+import svgToMiniDataURI from "mini-svg-data-uri";
+import { ComponentProps, memo, useMemo } from "react";
+import { ReactSVG } from "react-svg";
+
+import Icons from "./icons";
+
+export type Icons = keyof typeof Icons;
+
+export type Props = {
+  className?: string;
+  icon?: string;
+  height?: string | number;
+  width?: string | number;
+  alt?: string;
+  color?: string;
+  onClick?: () => void;
+};
+
+const Icon: React.FC<Props> = ({
```

Comment:
This component is copied from reearth-web before being modified, right? 
Please pull in the latest main before copying any code. This component was recently updated and doesn't use react-svg anymore